fastlane/Fastfile

```diff
@@ -549,17 +548,46 @@ ENV["APP_STORE_STRINGS_FILE_NAME"]="AppStoreStrings.pot"
       download_path: "./fastlane/metadata"
     )
 
-    # Create the marketing and privacy URL files
-    metadata_locales.each { |locale|
-      marketing_path = Dir.pwd + "/metadata/#{locale}/marketing_url.txt"
-      write_file(marketing_path, 'https://woocommerce.com/mobile/')
-      
-      privacy_path = Dir.pwd + "/metadata/#{locale}/privacy_url.txt"
-      write_file(privacy_path, "https://automattic.com/privacy/")
+    # Ensure that localizations are present in the right places (some locale codes aren't correct)
+    {
+      "ar" => "ar-SA",
+      "de" => "de-DE",
+      "es" => "es-ES",
+      "fr" => "fr-FR",
+      "nl" => "nl-NL",
+      "zh-cn" => "zh-Hans",
+      "zh-tw" => "zh-Hant"
+    }.each { |source, destination|
+      source_dir = Dir.pwd + "/metadata/" + source
+      Dir.children(source_dir).select { |file| 
+        FileUtils.mv source_dir + "/" + file, Dir.pwd + "/metadata/" + destination + "/" + file
+      }
```

Comment:
Also, why move each file in `source_dir` individually into `destination` manually?

Mhhh… Ok maybe I better do a more global suggestion then 😅 

```suggestion
    # Ensure that localizations are present in the right places (some locale codes aren't correct)
    YAML.load_file('locales.yml').each do |source, destination|
      source_files = Dir.glob(File.join('metadata', source, '*'))
      dest_dir = File.join('metadata', destination)
      FileUtils.mkdir_p dest_dir # In case it doesn't exist yet locally
      FileUtils.mv source_files, dest_dir
    end
```

This is assuming you also add the `locales.yml` YAML file
```yaml
ar: ar-SA
de: de-DE
es: es-ES
fr: fr-FR
nl: nl-NL
zh-cn: zh-Hans
zh-tw: zh-Hant
```

That last part is totally optional (it's ok if you prefer to keep the array hardcoded inline in the `Fastfile`) but if you go through with it it could be worth also updating `Scripts/update-translations.rb` to use it too (I think it's the same list right?)

Reply:
Yeah, see my note above – we could certainly do this, but IMHO it'd be nice to have a solution to this that's not on a per-project basis.

Reply:
Agreed you can skip over the YAML idea but the other refactorings (`File.join`, better use of `FileUtils`, …) in that suggestion are still worth applying even if you keep the list of locales hardcoded inline for now and that we do the rest later